### DISCUSSION TITLE
Cruiser 3.0.31

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#251] [Announcements]
Cruiser 3.0.31 is on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser):

- Display and announce nearby points of interest in navigation: fuel stations, speed cameras, ...
https://github.com/devemux86/cruiser/discussions/9

- Display and announce nearby favorites in navigation
https://github.com/devemux86/cruiser/discussions/209

- Search offline for POI categories
https://github.com/devemux86/cruiser/discussions/195

- BRouter (offline) new profile options: avoid cities / noise / traffic, prefer forests / rivers
https://github.com/devemux86/cruiser/discussions/248

<img src="https://github.com/devemux86/cruiser/assets/3484020/b8e632c2-85b0-4c9c-8c14-909af377e4e3" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/b0208eaa-a914-4e92-965c-787f830c78bd" width="50%"/>

**Comment by Leeb64** [comment]
When changing the application colors, the offline maps go away, along with the map style ????

**Reply by devemux86** [reply]
@Leeb64 it should be fixed in [Cruiser 3.0.28](https://play.google.com/store/apps/details?id=gr.talent.cruiser), please see:
- https://github.com/devemux86/cruiser/discussions/262

**Comment by r11d11** [comment]
Since update to 3..0.27 I have an issue with the font which is used to display  gps recording on the right site last .Looks like bold? Update to 3.0.28  did not make it better. See screenshot:
<img src="https://github.com/devemux86/cruiser/assets/101214119/41be73e6-2c44-4208-b7b6-05da0f8976ed" width="50%"/>

**Reply by devemux86** [reply]
@r11d11 thanks for the report.

This is strange, it is the same panel as before.

- Did you restart the app and remove it from Android recent apps screen?
- Restart your phone?
- What settings do you use in _Android Display settings_ for font + display size? 100% or something else?
- What _settings do you use in the app_ for font + display size?
- Can you attach the app settings .pref file?
"Settings | Application | Advanced | Preferences | Export"

**Reply by devemux86** [reply]
Something seems wrong on your screenshot, also the compass bearing "N" has very light color.

**Reply by r11d11** [reply]
Restarted the app, restart the phone but the font looks like in the screenshot. I only updated to 3.0.27 and did not change anything. Nore in Cruiser or in the settings of the phone.  Fontsize and display size is 100%. All other apps looks normal.  I found an workaround. Set my Xiaomi MI9 from Darkmode to Normal Mode in the settings of the phone.. Then the compass N and the gps recording display looks normal. Changed it back to darkmode and the problem returns. I have exported my settings:
[Cruiser 20230731_214703.zip](https://github.com/devemux86/cruiser/files/12221651/Cruiser.20230731_214703.zip)

**Reply by Ciutnik** [reply]
@r11d11 
Try to find the option to force the dark theme for individual apps in the MIUI settings, and disable it for Cruiser (By the way, it's best to turn it off for ALL apps, because it's very problematic).

**Reply by devemux86** [reply]
> Set my Xiaomi MI9 from Darkmode to Normal Mode in the settings of the phone.. Then the compass N and the gps recording display looks normal. Changed it back to darkmode and the problem returns.

@r11d11 thanks for the feedback, this seems to be the reason.
The forced dark mode by phones has not worked properly for many years.

@Ciutnik is right, it is always better to disable it.
At least Cruiser has its own light / dark app themes.

In any case I remember we had fixed this a long time ago.
Perhaps we need more checks in the code, I will see it.

**Reply by Ciutnik** [reply]
I just checked. 
Forced dark mode ON/OFF:

<img src="https://github.com/devemux86/cruiser/assets/122647782/e82ed6e6-e099-4f32-96b9-ccc20da33c57" width="49%"/> <img src="https://github.com/devemux86/cruiser/assets/122647782/17b9a101-e1aa-462f-bf0d-9102e441127d" width="49%"/>

Unfortunately, this crap is turned on by default for every newly installed application...

> In any case I remember we had fixed this a long time ago.

Yes, I remember too, I asked you for it and you handled it well 🙂

**Reply by devemux86** [reply]
@Ciutnik thanks for the test, so something has returned.

If anyone is interested, please contact me (email is on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser)).
I can send you a Cruiser apk to test on your devices.

**Reply by Ciutnik** [reply]
@devemux86 
Of course, I can check it on my old Mi8. But this is Android 10 (MIUI 12), although as I see similar problems.

**Reply by devemux86** [reply]
@r11d11 and @Ciutnik tested a Cruiser apk on their devices with a fix for the dark mode and the issue should be solved.

Thanks everyone for the report and the tests!

The fix will be included in the next Cruiser.

**Reply by r11d11** [reply]
I contacted devemux86, did a test with the brand new version and now the character set on cruiser with my MI9 in dark mode looks good again. I am happy. Thanks for the fast response :-)

**Reply by devemux86** [reply]
The system's dark mode on some phones has been fixed again in [Cruiser 3.0.29](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Comment by Leeb64** [comment]
Somewhat unfortunate German translation: Symbol Fork+Knive->> Lebensmittel.. would be better: Restaurant or Essen+Trinken.
Lebensmittel means a grocery store.

**Reply by devemux86** [reply]
@Leeb64 thanks for the report.

The English name is "Food".
What is the German translation? "Essen"?

Also the [Biker theme](https://github.com/mapsforge/vtm/blob/master/vtm-themes/resources/assets/vtm/biker.xml) has the same translation.

**Reply by Leeb64** [reply]
Only: "Essen"... you don't normally say it

"Speisen" or "Essen + Trinken" would be good. 

"Lebensmittel" means "Groceries" ....OK. that's what you eat... but it means something else.

**Reply by devemux86** [reply]
"Drink" is a separate category with translation "Getränk".
Is this correct?

**Reply by Leeb64** [reply]
Yes its correct.

**Reply by DolomitesTom** [reply]
The resulted search is a list of Pubs or Bistros. Therefore I would also call it "Bistro" in German language

**Reply by devemux86** [reply]
Thanks, so we just need to change the "Food" translation to "Speisen".

**Reply by Leeb64** [reply]
Or more generally: "Gaststätte"

**Reply by Leeb64** [reply]
I think "Speisen" is OK.

**Reply by DolomitesTom** [reply]
The resulted search are restaurants, Döner Grill and Imbiss. Gaststätten/Imbiss would be good.

**Reply by DolomitesTom** [reply]
Speisen would also work

**Reply by devemux86** [reply]
"Food" currently contains restaurants + fast food.
"Drink" currently contains bars + pubs.
More tags could be added to them, if we need them.

Please see all tags in this list:

- https://github.com/devemux86/cruiser/discussions/195#discussioncomment-6451058

**Comment by Leeb64** [comment]
How can you remove the POIs that are shown on the map?

**Reply by devemux86** [reply]
Long press on POI and select "Delete all".

**Reply by Leeb64** [reply]
...again something learned..thank you

**Comment by devemux86** [comment]
Next Cruiser will have a very significant improvement for all of us who use **Bluetooth** connection!

There will be much **improved voice volume** when we select to prepare the Bluetooth connection
before voice announcements, using the following Cruiser options in "Settings | Voice guidance":
- Voice volume
- Prepare audio connection (ms)

**Reply by Ciutnik** [reply]
Great! I recently changed my BT receiver which is a bit quieter than the previous one so I was looking for a solution to forcing the volume a bit higher than I can do in the system. 

I use the notification channel for the Cruiser audio (I'm getting blues on the media channel 😎). Will the upcoming change help me?

**Reply by devemux86** [reply]
@Ciutnik it will certainly help in the case you use the Cruiser option "Prepare audio connection".

Also try the Cruiser option "Voice volume" to set the voice volume independently from the system.

**Reply by devemux86** [reply]
> There will be much improved voice volume when we select to prepare the Bluetooth connection before voice announcements

An improved voice volume in Bluetooth connections is available in [Cruiser 3.0.31](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by Ciutnik** [reply]
It seems to me that "Prepare audio connection" helps with truncated messages(?) - I don't have this problem. Only I would like to set the volume of notifications to... 110%. Always, after detecting a motorcycle, Tasker set the volume of notifications to 100% and music to 80%, and that was fine. It's a little too quiet now (I still have some volume left for media but not notifications). Of course I also tried to force 100% in Cruiser, but it didn't help.

Some media players (MX Player, PowerAmp) have a kind of system-independent volume control and allow you to set the level well above what the system allows. I've seen apps that claim to do this but I haven't tested it yet. Also, I'm not sure if such tricks are allowed in the notification channel.

**Reply by devemux86** [reply]
> It seems to me that "Prepare audio connection" helps with truncated messages(?)

Correctly, as some Bluetooth devices need some ms to open.

> set the level well above what the system allows

Τhis is something worth checking if it is possible and how it can be done. :slightly_smiling_face:
